rules/linux/privilege_escalation_shadow_file_read.toml

```
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2022/09/01"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/09/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the manual reading of the /etc/shadow file via the commandline using standard system utilities.
+Threat actors will attempt to read this file, after elevating their privileges to root,
+in order to gain valid credentials they can utilize to move laterally undetected and access additional resources.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Shadow file read via commandline utils"
```

Can you update all these names to use title casing for consistency withe existing rules

```suggestion
name = "Shadow File Read via Command Line Utilities"
```